src/redux/uniqueTokens.js

```diff
@@ -117,8 +119,10 @@ export const fetchUniqueTokens = showcaseAddress => async (
           type: UNIQUE_TOKENS_GET_UNIQUE_TOKENS_SUCCESS,
         });
       }
-
       if (shouldStopFetching) {
+        const poaps = await fetchPoaps(accountAddress);
```

Comment:
what in case of failure? Did you test this?

Reply:
should just swallow, same as opensea. forgot to check tho @ibrahimtaveras00  is on it